tests/foreman/ui/test_dashboard.py

```diff
@@ -0,0 +1,485 @@
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier3
+    def test_positive_reset(self):
+        """@Test: Reset the Dashboard to default UI
+
+        @Feature: Dashboard
+
+        @Steps:
+
+        1.Navigate to Monitor -> Dashboard
+        2.Try to remove some widgets
+        3.Select the Manage Dropdown box
+        4.Save the Dashboard
+        5.Dashboard Widgets are saved successfully
+        6.Click Reset to default
+
+        @Assert: Widget positions successfully saved.
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier3
+    def test_positive_add_widgets(self):
+        """@Test: Add Widgets to Dashboard UI
+
+        @Feature: Dashboard
+
+        @Steps:
+
+        1.Navigate to Monitor -> Dashboard
+        2.Select Manage Dropdown box
+        3.Add Widgets
+
+        @Assert: User is able to add widgets.
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier3
+    def test_positive_search_bookmark(self):
+        """@Test: Bookmark the search filter in Dashboard UI
+
+        @Feature: Dashboard
+
+        @Steps:
+
+        1.Navigate to Monitor -> Dashboard
+        2.Add a filter to search box (eg. environment)
+        3.Bookmark the search filter
+
+        @Assert: User is able to list the Bookmark
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier3
+    def test_positive_host_configuration_status(self):
+        """@Test: Review the Host Configuration Status widget
+
+        @Feature: Dashboard
+
+        @Steps:
+
+        1.Navigate to Monitor -> Dashboard
+        2.Focus on the Host Configuration Status
+        3.Navigate to each of the links which has
+        search string associated with it.
+
+        @Assert: Each link shows the right info
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier3
+    def test_positive_host_configuration_chart(self):
+        """@Test: Check if Host Configuration Chart
+        widget is working in Dashboard UI
+
+        @Feature: Dashboard
+
+        @Steps:
+
+        1.Navigate to Monitor -> Dashboard
+        2.Review the Host Configuration Chart widget
+        3.Navigate to each of the links which
+        has search string associated with it.
+
+        @Assert: Each link shows the right info
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier3
+    def test_positive_task_status(self):
+        """@Test: Check if Task Status Dashboard UI
+        is displayed
+
+        @Feature: Dashboard
+
+        @Steps:
+
+        1.Navigate to Monitor -> Dashboard
+        2.Review the Task Status widget
+        3.Click each link
+
+        @Assert: Each link shows the right info
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier3
+    def test_positive_latest_warning_error_tasks(self):
+        """@Test: Check if Latest Warning/Error
+        Tasks widget is working in Dashboard UI
+
+        @Feature: Dashboard
+
+        @Steps:
+
```

Comment:
Ignore my comment.  Any multiline section should have a blank line for proper doc rendering.